### ISSUE TITLE
Notifications don't come

### ISSUE THREAD

**Issue by rulet** [k9mail/k-9#4448]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
Tell us what should happen
Notifications should come with sound and vibration on Android 9
### Actual behavior
Tell us what happens instead
Nothing
### Steps to reproduce
1.Just to send email to my gmail box
2.No notifications on blocked screen and on unblocked screen(app given all permissions including battery usage)
3.

### Environment
K-9 Mail version:5.703 

Android version:9

Account type (IMAP, POP3, WebDAV/Exchange):IMAP

Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here:
[k9-log.txt](https://github.com/k9mail/k-9/files/4054586/k9-log.txt)

**Comment by cketti** [comment]
Do new messages show up in the Inbox without manually refreshing the folder? If not, it's not a problem of missing notifications. It's that your Inbox isn't checked periodically. Make sure 'Manage folders > [Inbox] > Folder poll class' is set to '1st Class' and that 'Settings > [Account] > Fetching mail > Folder poll frequency' is set to something other than 'Never'.

**Comment by rulet** [comment]
I did what you said -- nothing changed. And minimum time for fetching poll frequency is too much -- 15min, I need at least 30sec. And, by the way, google services are not installed at all on the system.

**Comment by cketti** [comment]
You didn't answer the question. If new messages don't show up in the folder without manually refreshing, K-9 Mail doesn't fetch messages in the background (and hence can't know that notifications should be created). This would be a different problem than notifications not showing.
